compiler/optimizer/LoopReducer.cpp

```diff
@@ -1088,23 +1088,24 @@ TR_LoopReducer::generateArraycopy(TR_InductionVariable * indVar, TR::Block * loo
    if (arraycopyLoop.getStoreAddress()->getIncrement() < 0)
       //&& (storeAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol() == loadAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol()))
       {
-      if (comp()->cg()->getSupportsPrimitiveArrayCopy())
-         arraycopy->setBackwardArrayCopy(true); /* bit available only to primitive arraycopy */
```

Comment:
isBackwardArrayCopy is used in ValuePropagation.cpp:5518 ... still trying to work out if this code is limited in the cases it handles - it seems tied to the arraycopy implementations in the codegen...